djangocms_versioning_filer/monkeypatch/admin/admin_fields.py

```diff
@@ -0,0 +1,184 @@
+'''The monkeypatch done here allows you to save image plugins where the
+file is in draft mode'''
+
+import logging
+
+from django.contrib.admin.sites import site
+from django.contrib.admin.widgets import ForeignKeyRawIdWidget
+from django.db.models import ForeignKey
+from django.db.models.sql.where import WhereNode
+from django.dispatch import receiver
+from django.template.loader import render_to_string
+from django.urls import reverse
+from django.utils.http import urlencode
+from django.utils.safestring import mark_safe
+
+from cms.models.pluginmodel import CMSPlugin
+from cms.models.titlemodels import PageContent
+
+import filer
+from djangocms_versioning.constants import DRAFT, OPERATION_PUBLISH
+from djangocms_versioning.models import Version
+from djangocms_versioning.signals import post_version_operation
+from filer import settings as filer_settings
+from filer.fields.file import AdminFileFormField, FilerFileField
+from filer.models.filemodels import File
+
+
+logger = logging.getLogger(__name__)
+
+
+def render(self, name, value, attrs=None, renderer=None):
+    obj = self.obj_for_value(value)
+    css_id = attrs.get('id', 'id_image_x')
+    related_url = None
+    if value:
+        try:
+            # We need to get the original manager to be able to access drafts.
+            file_obj = File._original_manager.get(pk=value)
```

Comment:
It's a damn shame that we have to bring in the whole method for one line, shows that it is badly written in Filer.

Reply:
Yup, I don't think there is much we can do about that. If we don't modify filer in the first place. It would be reasonably easy to wrap the manager so only a tiny change is needed. So if you're able to pull some strings to get something like that accepted and a new version released by divio that might be viable solution. 

Otherwise this is what we have.